process/sync/metablock.go

```diff
@@ -511,3 +519,10 @@ func (boot *MetaBootstrap) getMetaHeaderFromPool(headerHash []byte) (data.Header
 func (boot *MetaBootstrap) getBlockBodyRequestingIfMissing(headerHandler data.HeaderHandler) (data.BodyHandler, error) {
 	return boot.getBlockBody(headerHandler)
 }
+
+func (boot *MetaBootstrap) isForkTriggeredByMeta() bool {
+	return boot.forkInfo.IsDetected &&
```

Comment:
this function should not be used in meta. forks inside meta is tiggered by meta :dagger:

Reply:
Exactly. This method is used in rollBack to know if the rollback header should be added in the black list or not. As now we have only one method rollBack for both meta/shard, isForkTriggeredByMeta method is implemented in shardBlock but also here, just to satisfy the interface blockBootstrapper and actually it will return always true when a fork in meta is detected, because also the following conditions would be true